gg_interface/src/main.cpp

```diff
@@ -39,21 +37,27 @@ void setup() {
     ina260.MaskEnable->read();
 
     // Enable alert on "Conversion Ready"
-    ina260.MaskEnable->write(0x0400);
+    ina260.MaskEnable->write(CONVERSION_READY_ADDRESS);
```

Comment:
Nitpick: 0x0400 is not a register address, it is an interrupt mask (or ALERT in the parlance of the INA260).

Reply:
Good call, I'll update the var name!